http/src/main/scala/org/broadinstitute/dsde/workbench/leonardo/util/GKEInterpreter.scala

```diff
@@ -968,14 +976,25 @@ class GKEInterpreter[F[_]: Parallel: ContextShift: Timer](
       _ <- logger.info(ctx.loggingCtx)(
         s"about to process descriptor for app ${appName.value} in cluster ${dbCluster.getGkeClusterId.toString} | trace id: ${ctx.traceId}"
       )
+      // Currently we always retry.
+      // The main failure mode here is helm install, which does not have easily interpretable error codes
+      retryConfig = RetryPredicates.standardRetryAllConfig
 
-      descriptor <- appDescriptorDAO.getDescriptor(desc).adaptError {
+      getDescriptor = appDescriptorDAO.getDescriptor(desc).adaptError {
         case e =>
           AppCreationException(
             s"Failed to process descriptor: $desc. Please ensure it is a valid descriptor, and that the remote file is valid yaml following the schema detailed here: https://github.com/DataBiosphere/terra-app#app-schema. \n\tOriginal message: ${e.getMessage}"
           )
       }
 
+      // Currently we always retry.
+      // This delays error reporting if the descriptor doesn't exist, but this shouldn't be a common failure mode except for very early in development
+      // We expect the majority of failures are due to intermittent network errors
+      descriptor <- tracedRetryF(retryConfig)(
```

Comment:
Posted in slack too :) Not sure we want to retry here since the `http4s` already retries exponentially 5 times?